docs/getting-started.md

```diff
@@ -79,34 +79,7 @@ printed next.
 This is the example that appears on the [main page][docs-main] of the documentation.
 
 ```python linenums="1"
-import columbo
-
-interactions = [
-    columbo.Echo("Welcome to the Columbo example"),
-    columbo.Acknowledge(
-        "Press enter to start"
-    ),
-    columbo.BasicQuestion(
-        "user",
-        "What is your name?",
-        default="Patrick",
-    ),
-    columbo.BasicQuestion(
-        "user_email",
-        lambda answers: f"""What email address should be used to contact {answers["user"]}?""",
-        default="me@example.com"
-    ),
-    columbo.Choice(
-        "mood",
-        "How are you feeling today?",
-        options=["happy", "sad", "sleepy", "confused"],
-        default="happy",
-    ),
-    columbo.Confirm("likes_dogs", "Do you like dogs?", default=True),
-]
-
-answers = columbo.get_answers(interactions)
-print(answers)
+{!.examples/index_user_prompts.py!}
```

Comment:
In this file and the other .md doc files, I've updated them as is done [in pydantic](https://github.com/samuelcolvin/pydantic/blob/d5e9d9abc8c8f554d660801ddc14bf6f32c9db45/docs/index.md#example).

Reply:
I'm not sure unfortunately. 🤔 I'll pull repo and run the `mkdocs` docker-compose and check it out as well.